lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Model for Course Card.
+ */
+(function (define) {
+    'use strict';
+    define([
+            'backbone'
+        ], 
+        function (Backbone) {
+        return Backbone.Model.extend({
+            initialize: function(data) {
+                if (data){
+                    this.set({
+                        name: data.display_name,
+                        courseStart: data.course_start,
```

Comment:
I have remove the course_card_model in my latest revision and it looks much cleaner. Why do we need to have a course card model in this PR? Can't we just initiate a default `Backbone.Model` object like what I did in my current change?